use-package.el

```diff
@@ -656,47 +656,40 @@ manually updated package."
 ;; :bind, :bind*
 ;;
 
-(defsubst use-package-is-sympair (x &optional allow-vector)
-  "Return t if X has the type (STRING . SYMBOL)."
+(defsubst use-package-is-pair (x car-pred cdr-pred)
```

Comment:
I tested `use-package-normalize-mode` (`:mode`, `:interpreter`) and `use-package-normalize-binder` (`:bind`, `:bind*`). Perhaps `use-package-normalize-binder` could use a few more tests though.